packages/snaps-simulator/src/features/simulation/sagas.ts

```diff
@@ -80,16 +80,20 @@ export function* initSaga({ payload }: PayloadAction<string>) {
 
   const srp: string = yield select(getSrp);
 
+  const sharedHooks = {
+    getMnemonic: async () => mnemonicPhraseToBytes(srp),
```

Comment:
Can this be covered?

Reply:
None of the hooks are currently covered because we don't have tests that run them directly. IMO out of scope for this PR and not exactly sure how we test to cover them